Cargo.toml

```diff
@@ -1,11 +1,10 @@
 [package]
 name = "hassle-rs"
 version = "0.1.7"
-authors = ["Jasper-Bekkers <bekkers@gmail.com>"]
+authors = ["Jasper-Bekkers <bekkers@gmail.com>", "Graham Wihlidal <graham@wihlidal.ca>"]
```

Comment:
I think we should add you prominently to the README.md as a significant contributor and leave me as the sole author for the crate for now.